srtcore/core.h

```diff
@@ -980,6 +980,10 @@ class CUDT
     CallbackHolder<srt_listen_callback_fn> m_cbAcceptHook;
     CallbackHolder<srt_connect_callback_fn> m_cbConnectHook;
 
+    void resetBufferWasFull() { 
+        m_bBufferWasFull = FALSE;
```

Comment:
`false`

Reply:
Another thing - why do you need this intermediate method here? You can just as well set this directly in `CUDT::startConnect`...

Reply:
True, I thought that going through a method was better than direct access, I'll update this

Reply:
The initialization is needed not just for SRT caller, but in general.